browser/lib/transport/xhrtransport.js

```diff
@@ -9,7 +9,7 @@ var XHRTransport = (function() {
 	XHRTransport.isAvailable = XHRRequest.isAvailable;
 
 	XHRTransport.checkConnectivity = function(callback) {
-		Http.Request('http://internet-up.ably.io.s3-website-us-east-1.amazonaws.com/is-the-internet-up.txt', null, null, null, function(err, responseText) {
+		Http.Request(Defaults.internetUpUrlWithoutExtension + '.txt', null, null, null, function(err, responseText) {
```

Comment:
@paddybyers can you confirm if this check is only made when in a browser?  If not, I will need to append the current protocol to this request when in a browser environment I assume.  Also, is internet connectivity only checked when falling back to XHR & JSONP? I assume that is intentional.